flang/docs/Extensions.md

```diff
@@ -170,6 +170,15 @@ end
   In the case of `DEFERRED` bindings in an `ABSTRACT` derived type,
   however, overrides are necessary, so they are permitted for inaccessible
   bindings with an optional warning.
+* Main program name is allowed to be the same as the other symbols used
+  in the main program, for example:
```

Comment:
Please add a note to the effect that the program name is capitalized, which is something that people using debuggers may need to know.